README.md

```diff
@@ -1,32 +1,86 @@
 # Ragbits
 
-Repository for internal experiment with our upcoming LLM framework.
+*A stack of ready-to-use libraries for building chatbots and other LLM-powered applications*
```

Comment:
Let's stick to the description form the repo - "Building blocks for rapid development of GenAI applications"

Reply:
Ok, I will, although personally I'm not a fun of marketing slogans that has a lot of buzzwords but fail to explain what the thing _actually_ is. Especially coupled with the fact that we don't have any longer explanation in this README, as a user I would just click away